File: CoffeeMaker/Java/src/main/java/WeakCoffee.java
import Interfaces.ICoffeeSelect;
import jdk.jshell.spi.ExecutionControl;

public class WeakCoffee implements ICoffeeSelect {

    public WeakCoffee()
    {
    }

    public int GetBeans() throws ExecutionControl.NotImplementedException {
        throw new ExecutionControl.NotImplementedException("Not implemented");
    }

    public boolean GrindBeans() throws ExecutionControl.NotImplementedException {
        throw new ExecutionControl.NotImplementedException("Not implemented");
    }
}

File: CoffeeMaker/CSHARP/CoffeeMaker/WeakCoffee.cs
using System;
namespace CoffeeMaker
{
    public class WeakCoffee : ICoffeeSelect
    {

        public WeakCoffee()
        {
        }

        public int GetBeans()
        {
            return 1;
        }

        public bool GrindBeans()
        {
            return true;
        }
    } 
    
}
